osf_tests/factories.py

```diff
@@ -465,8 +466,7 @@ def _create(cls, *args, **kwargs):
         registration.retract_registration(user)
         withdrawal = registration.retraction
         token = list(withdrawal.approval_state.values())[0]['approval_token']
-        with patch('osf.models.AbstractNode.update_search'):
```

Comment:
This isn't necessary anymore because the search is configured differently then it was when this was written.